frame/dapps-staking/src/lib.rs

```diff
@@ -796,12 +881,73 @@ impl<T: Trait> Module<T> {
                 }),
         }
     }
+
+    fn take_in_nominations(
+        stash: &T::AccountId,
+        nominations: Nominations<T::AccountId, BalanceOf<T>>,
+    ) {
+        if let Some(current_era) = T::EraFinder::current() {
+            let next_era = current_era + 1;
+
+            for (target, value) in nominations.targets.iter() {
+                if let Some(operator) = T::ContractFinder::operator(&target) {
```

Comment:
At the UI's request, it is necessary to obtain information about the user who is staking at a certain contract address.
So, EraStakingPoints should be [contract] => StakingPoints, as before.

It would be nice to introduce something like the new EraStakedOperator to have the sum of the staking for the contract held by the operator.

```
operator rewards
 = contract_1_staked/total_contract_staked + contract_2/total_contract_staked + ...
 = (contract_1_staked + contract_2_staked+ ...)/total_contract_staked.
```
```
[operator address] => this molecule
```
to lighten the information needed by the `claim_for_operator`.

Reply:
modified in 4c70c0b